server/sonar-web/src/main/js/apps/component-measures/components/bubbleChart/BubbleChart.js

```diff
@@ -106,6 +106,7 @@ export default class BubbleChart extends React.Component {
   }
 
   getTooltip(component) {
+    /* eslint-disable max-len */
```

Comment:
Ok I don't know, I tried in Atom and didn't get the warning for a test line I wrote after the comment in an other scope so I thought it wasn't block scoped.